src/pool-bin/libraries/BinPool.sol

```diff
@@ -385,12 +389,12 @@ library BinPool {
             amountsLeft = amountsLeft.sub(amountsIn);
             feeAmountToProtocol = feeAmountToProtocol.add(binFeeAmt);
 
+            shares = _addShare(self, params.to, id, params.salt, shares);
+
             arrays.ids[i] = id;
             arrays.amounts[i] = amountsInToBin;
             arrays.liquidityMinted[i] = shares;
```

Comment:
This is the liquidity added from user perspective, should we also mention it on IBinPoolManager that it could be less than the actual liquidity added into the pool from bin perspective ?